rfc0001.bs

```diff
@@ -57,17 +57,22 @@ This draft report is the work of the <a href="https://www.w3.org/community/web-a
 
 This extension has been created based on real-world use of metadata in the [[OpenAPI]], [[RAML]], [[API-Blueprint]], [[JSON-home]] and [[apis.json]] specifications.
 
-Definitions
-===========
+# Definitions
 
 <dfn>Machine-readable API Definition</dfn> a structured document defining the inputs and outputs of an API in a standardised format, primarily designed for automated processing, not human consumption.
 
 <div class="note">Note: See <a href="http://nordicapis.com/difference-api-documentation-specification-definition/">this article</a> for an explanation of the difference between API documentation, specifications and definitions.</div>
 
-Proposed Extension
-==================
+# Scope
 
-The base <a href="https://schema.org">Schema.org</a> <a href="https://pending.schema.org">WebAPI</a> type, for example:
+This specification does not attempt to replace existing formats of describing an API, such as [OpenAPI]], [[RAML]], [[API-Blueprint]], [[JSON-home]] and [[apis.json]], but rather seeks to describe metadata about an API that maybe useful to those discovering it.
+
+
+# Proposed Extension
+
+Schema.org's Dataset vocabulary was originally [based on DCAT](https://schema.org/docs/data-and-datasets.html), which in turn used used Dublin Core and FOAF terms. Hence it follows that [DCAT Version 2](https://www.w3.org/TR/vocab-dcat-2/) should be used to influence this extension.
```

Comment:
"Hence it follows" reads as a bit of a non-sequiteur. Can we say simply something along the lines that DCAT v2 is now current and is therefore used within this draft?

Reply:
Great feedback, have updated